{"name": "check users bills", "status": "broken", "statusDetails": {"message": "AttributeError: 'Context' object has no attribute 'fin_user'\n", "trace": "  File \"c:\\users\\wsu\\pycharmprojects\\msw_api\\venv\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"c:\\users\\wsu\\pycharmprojects\\msw_api\\venv\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\accounting_system.py\", line 66, in step_impl\n    session = context.fin_user\n  File \"c:\\users\\wsu\\pycharmprojects\\msw_api\\venv\\lib\\site-packages\\behave\\runner.py\", line 321, in __getattr__\n    raise AttributeError(msg)\n"}, "steps": [{"name": "Then check users bills", "status": "broken", "statusDetails": {"message": "AttributeError: 'Context' object has no attribute 'fin_user'\n", "trace": "  File \"c:\\users\\wsu\\pycharmprojects\\msw_api\\venv\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"c:\\users\\wsu\\pycharmprojects\\msw_api\\venv\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\accounting_system.py\", line 66, in step_impl\n    session = context.fin_user\n  File \"c:\\users\\wsu\\pycharmprojects\\msw_api\\venv\\lib\\site-packages\\behave\\runner.py\", line 321, in __getattr__\n    raise AttributeError(msg)\n"}, "start": 1610715002266, "stop": 1610715002267}], "start": 1610715002265, "stop": 1610715002267, "uuid": "ff4353ec-972d-4a40-98be-26ba4631beca", "historyId": "6d6a6330480bef18a36c0f6553303094", "fullName": "features/I-AS/II-fin_user.feature:check users bills", "labels": [{"name": "severity", "value": "normal"}, {"name": "tag", "value": "fin_user"}, {"name": "feature", "value": "FIN permissions"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}